File: demo/java-practise/src/main/java/com/grapecity/datavisualization/tests/tools/practise/models/ConfigOption.java
package com.grapecity.datavisualization.tests.tools.practise.models;

public class ConfigOption {
    private String source;
    private String target;

    public ConfigOption() {
        this.source = "../source";
        this.target = "../target";
    }

    public String getSource() {
        return this.source;
    }

    public String getTarget() {
        return this.target;
    }

    public void setSource(String value) {
        this.source = value;
    }

    public void setTarget(String value) {
        this.target = value;
    }

}

File: demo/csharp-practise/GrapeCity.DataVisualization.Practise/Options/ConfigOption.cs
namespace GrapeCity.DataVisualization.Practise.Options
{
    internal class ConfigOption : IConfigOption
    {
        public ConfigOption(ISourceOption source, ITargetOption target)
        {
            this.Source = source;
            this.Target = target;
        }

        public ISourceOption Source { get; private set; }

        public ITargetOption Target { get; private set; }
    }
}
